src/index.js

```diff
@@ -158,7 +158,12 @@ function levelIteratorToIterator (li) {
     next: () => new Promise((resolve, reject) => {
       li.next((err, key, value) => {
         if (err) return reject(err)
-        if (key == null) return resolve({ done: true })
+        if (key == null) {
```

Comment:
For completeness can you also implement the [`Generator.prototype.throw()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Generator/throw) method with similar logic?